src/User32.Desktop/User32+SafeWindowStationHandle.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SafeWindowStationHandle"/> nested type.
+    /// </content>
+    public static partial class User32
+    {
+        /// <summary>
+        /// Represents a Desktop handle that can be closed with <see cref="CloseWindowStation(IntPtr)"/>.
+        /// </summary>
+        public class SafeWindowStationHandle : SafeHandle
```

Comment:
Yes, I suppose a base class in Windows.Core is a good idea. Please feel free to file a PR for it.